frameworks/elastic/tests/config.py

```diff
@@ -10,12 +10,16 @@
 import sdk_utils
 
 PACKAGE_NAME = 'elastic'
+FOLDERED_SERVICE_NAME = '/test/integration/' + PACKAGE_NAME
```

Comment:
this is in two (at least?) tests now, factor it out into `sdk_folders.py` or the like?